api/types.go

```diff
@@ -86,3 +88,9 @@ type GetStrategyResponse struct {
 	Tokens    []GetStrategyToken `json:"tokens"`
 	Predicate string             `json:"strategy"`
 }
+
+type QueueItemResponse struct {
+	Done   bool               `json:"done"`
+	Error  error              `json:"error"`
+	Census *GetCensusResponse `json:"census,omitempty"`
```

Comment:
just `CensusResponse` or better something such as `CensusQueueResponse`.

Btw, does it actually need to be a pointer and to have omitempty? Try to use omitempty only when necessary.